client/document/index.jsx

```diff
@@ -122,6 +122,7 @@ class Document extends React.Component {
 						[ 'is-group-' + sectionGroup ]: sectionGroup,
 						[ 'is-section-' + sectionName ]: sectionName,
 						'is-white-signup': sectionName === 'signup',
+						'is-mobile-app-view': app.isMobileApp,
```

Comment:
In theory, it's possible that `context.app` won't be defined; should we tighten this up so it doesn't blow just in case?

Reply:
I think it would always be defined mostly since we use the app values in other parts of calypso. 
But I can add a check for it.